### ISSUE TITLE
Add makefile targets specific to Ray Bennett's requests

### ISSUE THREAD

**Issue by smalers** [OpenCDSS/cdss-app-statemod-fortran#47]
Ray Bennett has requested makefile targets that automatically copy the StateMod executable to `/C/cdss/bin` and also a way to save the full log without having to type the longer command line mentioned in the `make help`.

**Comment by smalers** [comment]
I added the `statemodrb` and `verycleanrb` targets to the makefile.  The latter saves to the log file `statemod-compile.log`, but unfortunately buffers output until it is done so there is no feedback while it runs.  To overcome this issue I created the `makeverycleanrb` script that Ray can run.  Unfortuantely, it won't have execute permissions so have to run the following on the command line:

```
chmod a+x makstatemodrb makeverycleanrb
```

Hopefully one of these solutions gives Ray what he wants.  I'll keep the issue open until I get feedback.
